korangar_networking/examples/ollama-chat-bot.rs

```diff
@@ -59,7 +59,7 @@ impl MessageHistory {
     }
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
```

Comment:
What is the reason for this change?

Reply:
We only use the single-threaded (ST) runtime in the project, so to not include the multi-threaded (MT) runtime also, we also use the ST variant here. But if you don't want to use the feature as state below, this can be returned to use the MT runtime.

Reply:
Okay, I like this :+1: